modules/cnv_viewing_guest_ip_addresses.adoc

```diff
@@ -0,0 +1,59 @@
+[[viewing-guest-ip-addresses]]
```

Comment:
Which assembly is this module used in?

Reply:
It will go in the User's Guide assembly, but I need to merge another PR first. I'll try to get in the habit of adding the metadata at the top...

Reply:
Ok. Yeah,that metadata is helpful.